packages/home-spa/src/res/static/microservices.json

```
@@ -2,35 +2,35 @@
     {
       "name": "Authentication",
       "description": "Adding authentication to your application is pice of cake. Just integrate with our API and focus on application development.",
-      "link": "/get-started/docs/microservices/authorization-service/",
+      "link": "/get-started/docs/microservices/authorization/",
       "icon": "res/img/auth.svg",
       "imgBgColor": "#F3FAF2"
     },
     {
       "name": "Feedback",
       "description": "Allows users to review and rate services and products provided by teams. Built into a power packed microservice so serve all.",
-      "link": "/get-started/docs/microservices/feedback-service/",
+      "link": "/get-started/docs/microservices/feedback/",
```

The previous one was correct. Just the authorization link was incorrect.
```suggestion
      "link": "/get-started/docs/microservices/feedback-service",
```